### PR TITLE
[IMP] [10.0] BR001208-T24350: Add field "Terms and Conditions"

### PR DESCRIPTION
**Pull request by JoJoJoJoJoJoJo**
add field `terms_and_conditions` and amend the printout

### CODE REVIEW

File: business_requirement/models/business.py [hunk 1]
```diff
@@ -67,6 +67,12 @@ def _get_default_company(self):
         states={'draft': [('readonly', False)],
                 'confirmed': [('readonly', False)]}
     )
+    terms_and_conditions = fields.Html(
+        'Terms and Conditions',
+        read_only=True,
```
Comment (elicoidal) [inline business_requirement/models/business.py]: why readonly?
Shouldnot we add copy=true as well?

File: business_requirement/views/business_view.xml [hunk 2]
```diff
@@ -63,6 +63,9 @@
                             <group string="Test Case">
                                 <field name="test_case" nolabel="1" default_focus="0"/>
```
Comment (elicoidal) [inline business_requirement/views/business_view.xml]: remove default_focus="0"

**Review by elicoidal — CHANGES_REQUESTED** [review]
LGTM: some details

Good job!

**Comment by JoJoJoJoJoJoJo** [comment]
@elicoidal 
changes done by your request.

File: business_requirement/models/business.py (hunk 1 above)
Comment (JoJoJoJoJoJoJo) [reply business_requirement/models/business.py]: default copy=True

**Review by elicoidal — CHANGES_REQUESTED** [review]
almost perfect!
On the functional tests:
* Only thing missing: the field should not be printed in the printout if it is empty.

**Comment by JoJoJoJoJoJoJo** [comment]
> Only thing missing: the field should not be printed in the printout if it is empty.

@elicoidal 
Any details about the test condition? I tested locally and it works normally

update:
I've found the situation, it seems when you create a BR, `<p><br></p>` will be added in the field, it's invisible but not empty

**Comment by elicoidal** [comment]
@JoJoJoJoJoJoJo I understand then it is LGTM!
Good job :)

**Review by elicoidal — APPROVED** [review]
LGTM

**Comment by elicoidal** [comment]
cc @victormartinelicocorp

**Review by lonelysun — APPROVED** [review]

**Review by martintorresvm — CHANGES_REQUESTED** [review]
bump module version, the rest LGTM

**Comment by JoJoJoJoJoJoJo** [comment]
@victormartinelicocorp 
version bumped
cc @elicoidal @lonelysun

**Review by martintorresvm — APPROVED** [review]

**Pull request merged**